src/Microsoft.Azure.WebJobs.Host/Bindings/BindingFactory.cs

```diff
@@ -176,7 +176,37 @@ public IBindingProvider BindToGenericItem<TAttribute>(Func<TAttribute, Type, Tas
             Func<TAttribute, ParameterInfo, INameResolver, Task<TAttribute>> postResolveHook = null)
             where TAttribute : Attribute
         {
-            var bindingProvider = new ExactTypeBindingProvider<TAttribute, TUserType>(_nameResolver, buildFromAttribute, buildParameterDescriptor, postResolveHook);
+            var bindingProvider = new ExactTypeBindingProvider<TAttribute, TUserType>(
+                _nameResolver, 
+                buildFromAttribute,
+                null,
+                buildParameterDescriptor, 
+                postResolveHook);
+            return bindingProvider;
+        }
+
+        /// <summary>
+        /// Create a binding provider that binds the parameter to a set of possible types via converter manager. 
```

Comment:
nit: can you use consistent capitalization in these blocks? e.g. some sentences start with capital, some don't, etc.